rust/core-lib/src/tabs.rs

```diff
@@ -550,6 +551,7 @@ impl CoreState {
     }
 
     // Detects indentation settings and returns an updated config if autodetect is on.
+    // todo: add line ending detection here as well
```

Comment:
Stale comment? The rest of the diff does seem to deal with line endings too.

Reply:
correct. I'll remove that in the next commit